miner/miner.go

```diff
@@ -192,3 +204,138 @@ func (miner *Miner) SetEtherbase(addr common.Address) {
 func (self *Miner) SubscribePendingLogs(ch chan<- []*types.Log) event.Subscription {
 	return self.worker.pendingLogsFeed.Subscribe(ch)
 }
+
+// commitmentCacheSaved will check to see if the last commitment cache entry is saved.
+// If not, it will search for it and save it.
+func (miner *Miner) commitmentCacheSaved(istEngine consensus.Istanbul) bool {
```

Comment:
function name suggest is just a query, no side effect or write. I would change the name to make that more apparent.